build/android/src/main/java/net.minetest.minetest/MtNativeActivity.java

```diff
@@ -19,6 +19,10 @@ public void onCreate(Bundle savedInstanceState) {
 	public void onDestroy() {
 		super.onDestroy();
 	}
+	
+	@Override
+	public void onBackPressed() {
```

Comment:
When does it happen? Do I have to be in a special menu? For me it shows the esc menu when inside the game when I press the back button, is it different for you?